target/linux/mediatek/image/filogic.mk

```diff
@@ -229,3 +229,26 @@ ifneq ($(CONFIG_TARGET_ROOTFS_INITRAMFS),)
 endif
 endef
 TARGET_DEVICES += xiaomi_redmi-router-ax6000-ubootmod
+
+define Device/Zyxel_EX5601-T0-stock
+  DEVICE_VENDOR := Zyxel
+  DEVICE_MODEL := EX5601-T0 (NAND)
```

Comment:
Does non NAND version of the EX5601-T0 also exist?

Reply:
Only nand is possible. I removed nand and added a reference to the stock partitioning